generatebundlefile/input.go

```diff
@@ -153,15 +153,20 @@ func (c *SDKClients) NewBundleFromInput(Input *Input) (api.PackageBundleSpec, st
 	if Input.MinVersion != "" {
 		packageBundleSpec.MinVersion = Input.MinVersion
 	}
+	copyImages := map[string]bool{}
 	for _, org := range Input.Packages {
 		for _, project := range org.Projects {
+			copyImages[project.Repository] = false
+			if project.CopyImages {
+				copyImages[project.Repository] = true
+			}
```

Comment:
```suggestion
			copyImages[project.Repository] = project.CopyImages
```

Reply:
Will this let the default be false? I can try this out.